index.html

```diff
@@ -213,19 +214,19 @@ <h2>LET’S CONNECT</h2>
                 </div>
               
                 <div class="container">
-  <form action="/action_page.php">
+  <form action="https://formspree.io/f/xeqdveok" method="post">
```

Comment:
- [ ] In terms of background images and general styling kindly stick to the Figma design. You will notice that the white spaces on both sides aren't equal too. Kindly fix it.

### Figma Design
![image](https://user-images.githubusercontent.com/86200356/200541847-398aa383-2911-40f2-bee7-ee5c9be52878.png)

### Your Design

### upper view
![image](https://user-images.githubusercontent.com/86200356/200543569-44fe9aa4-a7ba-4784-838c-39bdc865145b.png)

### lower view
![image](https://user-images.githubusercontent.com/86200356/200543356-ade26f32-a710-4efe-b8e2-cd3ec1692cbb.png)